### ISSUE TITLE
Keynav doesn't work occasionally.

### ISSUE THREAD

**Issue by numist** [numist/this-might-be-offensive#42]
Sometimes keyboard navigation doesn't work.

I think it's dependant on actions made while on the page, but I can't reproduce it (or remember what I did to trigger it in the first place).

Workaround is to click anywhere in the document and try again.

This regressed during a time when keyboard nav was made aware of modifier keys and when jQuery was updated from 1.4.2 to 1.7.1, so it's not obvious where this problem is coming from, but way that `handle_keypress` is bound to the event was not changed, and `handle_keypress` itself is pretty stable.

If anyone comes up with a reliable way to reproduce this on any platform, please update this issue.
